doc/engine/What's-new.md

```diff
@@ -1,190 +1,235 @@
+  subject to change.
+
+3. The new calculators can run with a single configuration file containing both
+the hazard and the risk parameters. This is actually the recommended way to
+run such calculators.
+
+6. The `scenario_risk` and `scenario_damage` calculators now support multiple
+GSIMs at the same time. This new functionality is documented in the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/).
+
+8. The `scenario_damage` calculator now supports multiple loss types at
+the same time, just as the `scenario_risk` calculator. Again, see the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/) for
+the details.
+
+7. The `event_based_risk` calculator has been optimized and enhanced. Now it
+is possible to generate the full event loss table for each
+asset and for each realization. Just set the configuration parameter
+`asset_loss_table=True` (the default is False).
+
+3. OpenQuake 1.6 supports officially the format NRML 0.5 for all risk
+models. In OpenQuake 1.5 it was supported in a limited and experimental way
+only for a subset of the vulnerability functions. Now all kind of risk
+models are supported: vulnerability models, fragility risk models and
+consequence models. Consequence models are brand new, introduced for
+the first time in this release. All of that is documented in the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/).
+The new format is simpler than before
+and more convenient to use, since the OpenQuake platform offers a web
+tool to prepare risk models in NRML 0.5. Beware that the web tool does
+not support validation of the risk models yet.
+
+4. The validation of the risk models in the engine has been
+improved. In particular now an user confusing a fragility model with a
+vulnerability model or a consequence model or any other combination
+will get a clear error message. Moreover, each risk model has a
+`lossCategory` attribute which must be set consistently with the name
+of the key in the job.ini file (see the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/)
+for the details).
+
+6. Some work has been going on hazardlib, as usual, and you can
+have a look at the [changelog](https://github.com/gem/oq-hazardlib/blob/engine-1.6/debian/changelog). The most prominent feature is the introduction of
+epistemic uncertainties on the fault geometry into the Source Model Logic Tree.
+
+7. We added an optional attribute `discretization` to the
+area source geometry XML description: this means that it is possible to
+specify a source-specific discretization step. This is useful in site specific
+analysis: area sources with little impact on the site of interest can use
+a large discretization step whereas the important area sources can use a
+finer discretization step.
+
+10. Two new GMPEs have been added to hazardlib: *Montalva et al. (2015)*
+and *Tusa and Langer (2015)*.
```

Comment:
If this PR (https://github.com/gem/oq-hazardlib/pull/394) gets merged in time then you can add *Allen, Wald and Worden (2012) (Intensity Prediction Equation)* to that list